products/memgraph/v0.15.0/reference_guide/other-features.md

```diff
@@ -279,14 +279,14 @@ section](#inspecting-queries), it is also possible to profile the execution of a
 query and get a detailed report on how the query's plan behaved. For every
 logical operator the following info is provided:
 
-- `ACTUAL HITS` -- the number of times a particular logical operator was pulled
-  from
+- `ACTUAL HITS` --- the number of times a particular logical operator was pulled
```

Comment:
This is also not rendered correctly :)
You need to type `&mdash;` verbatim. (I guess you didn't see that because comments are also markdown).

Regarding the en dash vs em dash conundrum, em dashes should be correct here, although I agree they sometimes look ugly.

Reply:
Hopefully it's fixed now.